platforms/shared/configuration/roles/setup/vault_kubernetes/tasks/main.yaml

```diff
@@ -61,44 +61,6 @@
     KUBECONFIG={{ kubernetes.config_file }} kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " "
   register: kubernetes_server_url
 
-# Check if ClusterRoleBinding created
-- name: "Checking for the {{ component_ns }}-role-tokenreview-binding"
-  include_role:
-    name: "{{ playbook_dir }}/../../shared/configuration/roles/check/k8_component"
-  tags:
-    - notest
```

Comment:
Are these deletions necessary?

Reply:
Had to remove the checks, as the problem is with corda platform components (doorman, nms and notary) has there own vault auth policy as they have different secret engine path but they belong to same org. So if we keep the check doorman vault k8s job will create service account and clusterrole binding and then nms and notary jobs will be skipped as it finds the sa and clusterrole already present.

Reply:
Alternative approaches would be to have one vault auth policy by having these components under one secret engine path and run vault k8s job one time for the org, or create different service account and clusterrolebinding for doorman, nms and notary

Reply:
Update: Have added the checks back, as it was used in the task variables, have removed the when condition.